### ISSUE TITLE
Civic Tech Jobs: Update alt text for Civic Tech Jobs image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3950]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Civic Tech Jobs` image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/civic-tech-jobs.md`:

**From:**
```
Hack for LA logo and the title of the project, Civic Tech Jobs
```

**To:**
```
Civic Tech Jobs
```

**For PM, Merge Team, or Tech Lead**
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- Civic Tech Jobs: https://www.hackforla.org/projects/civic-tech-jobs

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/civic-tech-jobs.md`
- Click on the filename below to see the source file code: https://github.com/hackforla/website/blob/gh-pages/_projects/civic-tech-jobs.md

- This issue is part of epic #3866 

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by mjh-projects** [comment]
Availability for this week: Weekdays  after 5:30 PM 
Had some issues with computer so I am chaning the ETA: 
My estimated ETA for completing this issue: Tuesday (12/16) 10PM

**Comment by mjh-projects** [comment]
I am currently using a an ubuntu linux machine and I am having trouble getting docker to work. I am not sure if I need to have additional permissons setup. I get the following error when trying to start Docker: 

```
ubuntu@ubuntu-Cyan:~/hackforla/website$ docker-compose up
Creating hfla_site ... done
Attaching to hfla_site
hfla_site    | ruby 2.7.4p191 (2021-07-07 revision a21a3b7d23) [x86_64-linux-musl]
hfla_site    | Configuration file: /srv/jekyll/_config.yml
hfla_site    |            Cleaner: Nothing to do for /srv/jekyll/_site.
hfla_site    |            Cleaner: Nothing to do for ./.jekyll-metadata.
hfla_site    |            Cleaner: Nothing to do for .sass-cache.
hfla_site    | ruby 2.7.4p191 (2021-07-07 revision a21a3b7d23) [x86_64-linux-musl]
hfla_site    | Configuration file: _config.yml
hfla_site    | Configuration file: _config.docker.yml
hfla_site    |             Source: .
hfla_site    |        Destination: /srv/jekyll/_site
hfla_site    |  Incremental build: enabled
hfla_site    |       Generating... 
hfla_site    | jekyll 3.9.2 | Error:  Permission denied @ dir_s_mkdir - /srv/jekyll/_site
hfla_site    | /usr/local/lib/ruby/2.7.0/fileutils.rb:250:in `mkdir': Permission denied @ dir_s_mkdir - /srv/jekyll/_site (Errno::EACCES)
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:250:in `fu_mkdir'
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:228:in `block (2 levels) in mkdir_p'
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:226:in `reverse_each'
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:226:in `block in mkdir_p'
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:211:in `each'
hfla_site    | 	from /usr/local/lib/ruby/2.7.0/fileutils.rb:211:in `mkdir_p'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/convertible.rb:226:in `write'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:209:in `block in write'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:332:in `block (2 levels) in each_site_file'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:331:in `each'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:331:in `block in each_site_file'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:330:in `each'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:330:in `each_site_file'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:208:in `write'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:73:in `process'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/command.rb:28:in `process_site'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/build.rb:65:in `build'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/build.rb:36:in `process'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/serve.rb:93:in `block in start'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/serve.rb:93:in `each'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/serve.rb:93:in `start'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
hfla_site    | 	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
hfla_site    | 	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
hfla_site    | 	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
hfla_site    | 	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
hfla_site    | 	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
hfla_site    | 	from /usr/gem/gems/jekyll-3.9.2/exe/jekyll:15:in `<top (required)>'
hfla_site    | 	from /usr/gem/bin/jekyll:25:in `load'
hfla_site    | 	from /usr/gem/bin/jekyll:25:in `<main>'
hfla_site exited with code 1
```

**Comment by mjh-projects** [comment]
Seems it was a permissions issue with my install thanks @MattPereira it worked fine when I updated to v2 as suggested in Slack. Blocker solved!

Updated ETA 2/26 8 AM

**Comment by LOSjr4** [comment]
Availability for this week: M-F 9AM-1PM
My estimated ETA for completing this issue: today 03/16/23

**Comment by jdingeman** [comment]
@LOSjr4 - you do not need to manually close an issue that would be completed with a pull request. Is there a reason you assigned yourself to the issue when it already had an assignee?

**Comment by mjh-projects** [comment]
@jdingeman I was going to finish this issue as I synched my Github and just need to push the changes and submit for merge approval but it was closed. I am not sure how to proceed.
